src/containerapp/azext_containerapp/tests/latest/test_containerapp_session_code_interpreter.py

```diff
@@ -0,0 +1,109 @@
+
+        # List Session Pools
+        sessionpool_list = self.cmd("containerapp sessionpool list -g {}".format(resource_group)).get_output_in_json()
+        self.assertTrue(len(sessionpool_list) == 0)
+
+        # Create JupyterPython SessionPool
+        sessionpool_name_python = self.create_random_name(prefix='spjupyterpython', length=24)
+        self.cmd('containerapp sessionpool create -g {} -n {} --cooldown-period {}'.format(
+            resource_group, sessionpool_name_python, 300), checks=[
+            JMESPathCheck('name', sessionpool_name_python),
+            JMESPathCheck('properties.containerType', "PythonLTS"),
+            JMESPathCheck('properties.provisioningState', "Succeeded"),
+            JMESPathCheck('properties.dynamicPoolConfiguration.cooldownPeriodInSeconds', 300)
+        ])
+
+        # List Session Pools
+        sessionpool_list = self.cmd("containerapp sessionpool list -g {}".format(resource_group)).get_output_in_json()
+        self.assertTrue(len(sessionpool_list) == 1)
+
+        # execute python code
+        identifier_name = self.create_random_name(prefix='testidentifier', length=24)
+        code = '\'print("'"Hello world"'")\''
+        self.cmd("containerapp session code-interpreter execute -n {} -g {} --identifier {} --code {}".format(
+            sessionpool_name_python,
+            resource_group,
+            identifier_name,
+            code),
+            checks=[
+            JMESPathCheck('properties.status', 'Success'),
+            JMESPathCheck('properties.stdout', 'Hello world\n')
+        ])
+
+        # upload a file also add session pool location
+        txt_file = os.path.join(TEST_DIR, 'cert.txt')
+        self.cmd("containerapp session code-interpreter upload-file -n {} -g {} --identifier {} --filepath {} --session-pool-location {}".format(
+            sessionpool_name_python,
+            resource_group,
+            identifier_name,
+            txt_file,
+            TEST_LOCATION
+            ),
+            checks=[
+            JMESPathCheck('value[0].properties.filename', 'cert.txt'),
+        ])
+
+        # list files
+        files_list = self.cmd("containerapp session code-interpreter list-files -n {} -g {} --identifier {}".format(
+            sessionpool_name_python,
+            resource_group,
+            identifier_name
+            )).get_output_in_json()
+        self.assertTrue(len(files_list["value"]) == 1)
+
+        # check content
+        file_content = self.cmd("containerapp session code-interpreter show-file-content -n {} -g {} --identifier {} --filename {}".format(
+            sessionpool_name_python,
+            resource_group,
+            identifier_name,
+            "cert.txt"
+            )).get_output_in_json()
+        self.assertTrue(file_content == '\"testing\"')
+
+        # delete file
+        self.cmd("containerapp session code-interpreter delete-file -n {} -g {} --identifier {} --filename {} --yes".format(
```

Comment:
I checked the recording file, why the delete return `204`? Is the delete name correct?
```
- request:
    body: null
    headers:
      Accept:
      - '*/*'
      Accept-Encoding:
      - gzip, deflate
      CommandName:
      - containerapp session code-interpreter delete-file
      Connection:
      - keep-alive
      Content-Length:
      - '0'
      ParameterSetName:
      - -n -g --identifier --filename --yes
      User-Agent:
      - python/3.10.12 (Linux-5.15.146.1-microsoft-standard-WSL2-x86_64-with-glibc2.35)
        AZURECLI/2.60.0
    method: DELETE
    uri: https://northcentralusstage.dynamicsessions.io/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/clitest.rg000001/sessionPools/spjupyterpython000004/files/cert.txt?identifier=testidentifier000005&api-version=2024-02-02-preview
  response:
    body:
      string: ''
    headers:
      connection:
      - keep-alive
      date:
      - Wed, 15 May 2024 01:20:02 GMT
      mise-correlation-id:
      - 05a9aab5-c531-439d-bdb5-c92e9b673461
      server:
      - Microsoft-IIS/10.0
      x-powered-by:
      - ASP.NET
    status:
      code: 204
      message: No Content
```

Reply:
delete name is correct for delete api in backend we return 204 if succeed
![image](https://github.com/Azure/azure-cli-extensions/assets/42949747/b2fc148a-b1d1-420e-84d1-e7bbfe004b03)